mods/ra2/rules/proxy-actors.yaml

```diff
@@ -5,3 +5,4 @@ barracks.infiltrated:
 warfactory.infiltrated:
 	AlwaysVisible:
 	ProvidesPrerequisite:
+
```

Comment:
Let's be careful not to go back and forth on these. How about we add them as the utility does, but never remove them?